.env.sample

```
@@ -34,7 +34,8 @@ OVH_ROOM_PHONE_NUMBER=CHANGE_ME
 
 # HTTP
 #PORT=8080
-PROTOCOL="http"
+# Hostname : the links sent in email will use this hostname. Default is http://localhost:$PORT
+HOSTNAME_WITH_PROTOCOL="https://myapp.com"
```

```suggestion
#HOSTNAME_WITH_PROTOCOL="https://myapp.com"
```
ou sinon mettre une valeur qui marche en local